packages/grafana-ui/src/themes/_variables.dark.scss.tmpl.ts

```diff
@@ -187,6 +187,8 @@ $btn-drag-image: '../img/grab_dark.svg';
 
 $navbar-btn-gicon-brightness: brightness(0.5);
 
+$btn-active-box-shadow: 0px 0px 4px rgba(255,120,10,1);
```

Comment:
`$btn-active-box-shadow` has been created for both themes and is then used in `explore-active-button-glow` class. This class is then used to create active button glow for all buttons, that have active state - timepicker, time-sync button, live-tailing button.